router/v3/channels.go

```diff
@@ -418,3 +418,14 @@ func (h *Handlers) GetChannelPath(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, echo.Map{"path": channelPath})
 }
+
+// GetDMChannelList GET /users/:userID/dm-channel-list
```

Comment:
基本的に自分の DM の履歴しか取得しない (というか、他人のが見られるとまずい) ので、 `/me` 以下で問題ないと思います！
```suggestion
// GetDMChannelList GET /users/me/dm-channel-list
```